packages/client/yggdrasil/yggdrasil.spec

```diff
@@ -0,0 +1,96 @@
+%define debug_package %{nil}
+
+%global go_arches x86_64 s390x ppc64le
```

Comment:
This is already defined, at least on EL7 (if golang is installed):
```console
# rpm --showrc |grep go_arc
-14: go_arches	%{ix86} x86_64 %{arm} aarch64 ppc64le
# grep go_arches /usr/lib/rpm/macros.d/macros.*
/usr/lib/rpm/macros.d/macros.golang:# ExclusiveArch:  %{go_arches}
/usr/lib/rpm/macros.d/macros.golang:%go_arches       %{ix86} x86_64 %{arm} aarch64 ppc64le
```

On EL you need to depend on `go-srpm-macros` for it. It's odd that https://github.com/RedHatInsights/yggdrasil/blob/main/yggdrasil.spec.in doesn't list this though.

Is this something we'd like to address?

Reply:
I'd prefer to stay as close to upstream spec as possible.

If it builds, it builds.

Reply:
Upstream does not declare it, which also means it can't build on EL8 as I understand it.

Reply:
I'd go with if it builds, it builds. I'd like to avoid the spec from getting more complex by pulling different deps on different platforms.